troubleshooting/general-issues.md

```diff
@@ -128,6 +129,10 @@ There are also certain measures a web app should take to improve its performance
 
 * If you need to generate documents or prepare some data, try to offload this task to the server. You can create an HTTP request and gather the processed information. With this approach, you will gain from the better server-side performance and won't download all the data on the client-side. You can find an example of the approach in the <a href="https://github.com/telerik/blazor-ui/tree/master/grid/pdf-export-server" target="_blank">Export Grid to PDF on the Server</a> sample project.
 
+## Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...
+
+The error is a [general Blazor WebAssembly issue]({%slug common-kb-failed-to-find%}), which can result from the build process, publishing process, or browser cache. There are online documentation and discussions on the matter, which suggest possible solutions.
```

Comment:
Reword the link text, so that it's more obvious it points to a related solution.

```suggestion
The error is a [general Blazor WebAssembly issue]({%slug common-kb-failed-to-find%}), which can result from the build process, publishing process, or browser cache. There are online discussions and documentation on the matter, which suggest possible solutions.
```